jenkins/shared-library/vars/configureArtifactRepository.groovy

```diff
@@ -0,0 +1,109 @@
+void call(String environment) {
+	final String ownerName = helper.resolveOrganizationName()
+	logger.logInfo("Configuring respository pull for ${environment}")
+
+	String url = resolveRepositoryUrl(ownerName, resolveRepositoryName(environment))
+
+	if (null != url) {
+		configure(environment, ownerName, url)
+	}
+}
+
+String resolveRepositoryName(String environment) {
+	Map<String, List<Object>> environmentRepoNameMap = [
+		'javascript' : 'npm-group',
+		'python': 'pypi-group'
+	]
+
+	return environmentRepoNameMap[environment] ?: ''
+}
+
+String readNpmPackageScopeName() {
+	Object packageJson = readJSON file: 'package.json'
+	String[] nameParts = packageJson.name.tokenize('/')
```

Comment:
why do you need to tokenize here? aren't names like `symbol-sdk`?

Reply:
no, their packages have a scope(triton), e.g.  ``@triton/venues``